exception-handling/README.md

```diff
@@ -0,0 +1,46 @@
+# Exception Handling With `IExceptionHandler`
```

Comment:
let's not rush this subject and dive into below document to have a deeper understanding of microsoft's error handling suggestions.

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-8.0